community.zh.md

```diff
@@ -7,31 +7,31 @@ title: "社区 & 项目信息"
 {% toc %}
 
 
-There are many ways to get help from the Apache Flink community. The [mailing lists](#mailing-lists) are the primary place where all Flink committers are present. For user support and questions use the *user mailing list*. Some committers are also monitoring [Stack Overflow](http://stackoverflow.com/questions/tagged/apache-flink). Please remember to tag your questions with the *[apache-flink](http://stackoverflow.com/questions/tagged/apache-flink)* tag. Bugs and feature requests can either be discussed on the *dev mailing list* or on [Jira]({{ site.jira }}). Those interested in contributing to Flink should check out the [contribution guide](how-to-contribute.html).
+我们可以通过多种形式从 Apache Flink 社区获得帮助。Flink commiter 主要活跃在 [邮件列表](#mailing-lists)。对于用户支持和问题咨询，则可以通过 *用户邮件列表* 获得帮助。有些 Committer 同时会关注 [Stack Overflow](http://stackoverflow.com/questions/tagged/apache-flink)。请在提问的时候记得添加 apache-flink 的标签。问题反馈以及新特性的讨论则可以在 *开发邮件列表* 或者 [Jira]({{ site.jira }}) 上进行讨论。有兴趣对 Flink 进行贡献的人请查阅 [贡献指南](how-to-contribute.html)。
 
-If you send us an email with a code snippet, make sure that:
+如果你的邮件中包含代码，请根据如下要求进行检查：
 
-1. you do not link to files in external services as such files can change, get deleted or the link might break and thus make an archived email thread useless
-2. you paste text instead of screenshots of text
-3. you keep formatting when pasting code in order to keep the code readable
-4. there are enough import statements to avoid ambiguities
+1. 请确认你所使用的外链不会被修改、删除等，这些操作会使得归档后的邮件变得无效
+2. 粘贴文字而不是文字的截图
+3. 对代码进行格式化操作，以提高可读性
+4. 添加足够的上下文，确保代码没有模棱两可的地方
 
 
-## Mailing Lists
+## 邮件列表
 
 <table class="table table-striped">
   <thead>
-    <th class="text-center">Name</th>
-    <th class="text-center">Subscribe</th>
-    <th class="text-center">Digest</th>
-    <th class="text-center">Unsubscribe</th>
-    <th class="text-center">Post</th>
-    <th class="text-center">Archive</th>
+    <th class="text-center">名字</th>
+    <th class="text-center">订阅</th>
+    <th class="text-center">摘要</th>
+    <th class="text-center">退订</th>
+    <th class="text-center">发送</th>
+    <th class="text-center">归档</th>
   </thead>
   <tr>
     <td>
       <strong>news</strong>@flink.apache.org<br>
-      <small>News and announcements from the Flink community</small>
+      <small>Flink 社区的新闻和公告</small>
```

Comment:
Translate "Subscribe" , "Unsubscribe", "Read only list", "Archives" in the table?